Assembler.Core/Entities/BaseFrame.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using Assembler.Core.Enums;
+
+namespace Assembler.Core.Entities
+{
+    public abstract class BaseFrame : BaseMessage
```

Comment:
Would it be better if I called it `BaseGuidMessage`?

Reply:
No, because it has no meaning. You dont use that type in the code as polymorphism, not even single time, so there is no reason for it to exist.

You can delete that type, and just have public Guid in the BaseFrame and BaseMessageInAssembly. Altough its (small) copy of code, its much better

Reply:
f